### PR TITLE
Problem: cosmos-sdk 0.46 is not used

### PR DESCRIPTION
**Pull request by yihuang**

### CODE REVIEW

File: orchestrator/cosmos_gravity/src/send.rs [hunk 1]
```diff
@@ -126,16 +126,20 @@ pub async fn send_messages(
...
     let cosmos_address = cosmos_key.to_address(&contact.get_prefix()).unwrap();
+    let min_gas_limit = 500000 * messages.len() as u64;
 
+    // compute the fee as fee=ceil(gas_limit * gas_price)
+    let fee_amount: f64 = min_gas_limit as f64 * gas_price.0;
+    let fee_amount: u64 = fee_amount.abs().ceil() as u64;
     let fee_amount = Coin {
         denom: gas_price.1.clone(),
-        amount: 0u32.into(),
+        amount: fee_amount.into(),
```
Comment (yihuang) [inline orchestrator/cosmos_gravity/src/send.rs:136]: cosmos-sdk 0.46 reject 0 gas tx.

Comment (yihuang) [reply orchestrator/cosmos_gravity/src/send.rs]: reverted the change, will be fixed at cosmos-sdk side: https://github.com/cosmos/cosmos-sdk/pull/12590

**Comment by EricBolten** [comment]
I know this is a draft PR, but just commenting: ibc-go 4 is still in alpha and 0.46 was just released. We'll upgrade to ibc-go 3 and 0.45.6 in the next release and wait on 0.46 a bit I imagine.

**Comment by EricBolten** [comment]
Closing this as we are managing the SDK update process -- the next version will have us on 0.45.13 and we will proceed to 0.46 at a later date.

**Pull request closed without merging**